### PR TITLE
Fix for PHP Notice: Undefined variable: search_type

### PR DESCRIPTION
**Pull request by ericjuden**
I was getting a PHP notice that search_type was undefined.

### CODE REVIEW

File: material/linklist.html [hunk 1]
```diff
@@ -16,21 +16,25 @@
         <div class="text-center">
             <img src="images/sad_star.png" alt="Nothing found" />
         </div>
-        <div class="nothing-found">Sorry... We found nothing{if="$search_type=='fulltext'"} for <strong>{$search_crits}</strong>{elseif="$search_type=='tags'"} for<strong>{loop="search_crits"} {$value}{/loop}</strong>{else}{/if}.</div>
+        <div class="nothing-found">Sorry... We found nothing{if="isset($search_type)"}{if="$search_type=='fulltext'"} for <strong>{$search_crits}</strong>{elseif="$search_type=='tags'"} for<strong>{loop="search_crits"} {$value}{/loop}</strong>{else}{/if}{/if}.</div>
     {else}
```
Comment (nicolasdanelon) [inline material/linklist.html]: can work something like `{if="isset($search_type) && $search_type=='fulltext'"}`

**Comment by ArthurHoaro** [comment]
`$search_type` doesn't exist anymore. Please use [`$search_term` and `$search_tags`](https://github.com/shaarli/Shaarli/blob/master/tpl/linklist.html#L48) instead.

**Comment by kalvn** [comment]
Sorry I close your PR, I implemented things a bit differently in my last commit: https://github.com/kalvn/Shaarli-Material/commit/ed935187692ef81f605b5e047d28532fe4988315

But thanks a lot anyway for your work!

**Comment by ericjuden** [comment]
No worries

**Pull request closed without merging**